CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.23.2 (Mar 16, 2022)
+
+IMPROVEMENTS:
+
+* resource/artifactory_remote_docker_repository: Setting default value '**' for external_dependencies_patterns field. [GH-363]
```

Comment:
Hmmm, there is a local v1 and local v2 docker repo. Are you sure you don't need something identical for remote?

Reply:
For docker remote repos ... only v2 is present.